### ISSUE TITLE
Green label component is not accessible

### ISSUE THREAD

**Issue by paulcpederson** [Esri/calcite-web#969]
Green labels don't pass WCAG AA compliance. We should use `$dark-green`. Related to #964 if there are new colors they may have an accessible green-white combination.
